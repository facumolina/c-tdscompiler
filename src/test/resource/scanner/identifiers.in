/* Identifiers */
asdasd
Hasjln2
aa22ss
sadasd_asda

/* Incorrect Identifiers */
23ab
_abc
{abc