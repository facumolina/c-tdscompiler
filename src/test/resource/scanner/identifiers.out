COMMENT
ID
ID
ID
ID
COMMENT
INT_LITERAL
ID
Illegal symbol, '_'
ID
L_BRACE
ID
